guides/common/modules/ref_performing-a-fresh-installation-of-a-server-on-rhel8.adoc

```diff
@@ -0,0 +1,7 @@
+[id="Performing_a_Fresh_Installation_of_a_Server_on_rhel8_{context}"]
+= Performing a Fresh Installation of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you have created a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 source machine, you can install {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 target machine.
+
+* To install {ProjectServer}, see xref:accessing-server[Accessing {ProjectServer}].
```

Comment:
This xref is broken:

```
$ head guides/common/modules/con_accessing-server.adoc
[id="Accessing_Server_{context}"]
= Accessing {ProjectName}
```

```suggestion
* To install {ProjectServer}, see xref:Accessing_Server_{context}.
```

Note that this only works if both files are in the same guide (which I did not check!)